### DISCUSSION TITLE
Do you think a postinstall script should be included?

### DISCUSSION THREAD

**Discussion by flexbox** [mmazzarolo/react-native-universal-monorepo#44] [Polls]
**Motivation**
Improving the DX setup

**Scenario**
1. I am cloning the project on macOS
2. install all the dependencies
3. run `yarn ios:start`

```
error Failed to build iOS project. We ran "xcodebuild" command but it exited with error code 65.
```
😭

Obviously, I forgot to run pod install _—like every time_ 😅

The folks at infinite.red have a really nice `postInstall` script that solves this issue
https://github.com/infinitered/ignite/blob/master/boilerplate/bin/postInstall

**Comment by mmazzarolo** [comment]
Hola! I feel it would be better to keep each "app" as close as possible to the barebone version provisioned with `react-native init`. (don't get me wrong, I also think it's better to have a `postInstall` script in general, but it's not something I would be opinionated about in this repo). 
But I'm open to do it if I see enough requests for it 👍

**Reply by flexbox** [reply]
@mmazzarolo I agreed with you :) but in the meantime, I like to automate stuff 

Let's the maths decide if people find it useful or not
